### PR TITLE
megan added a draft tutorial for the core-dataset tutorial

### PR DESCRIPTION
**Pull request by meganmason**
The directions worked well to add a tutorial. Now I need to go back and put some real meat in the tutorial.

### CODE REVIEW

File: book/_toc.yml [hunk 1]
```diff
@@ -23,5 +23,6 @@
 - file: tutorials/index
   sections:
     - file: tutorials/raster/intro-ipyleaflet
+    - file: tutorials/core-datasets/core-datasets-tutorial
```
Comment (scottyhq) [inline book/_toc.yml:26]: thanks @meganmason. looks like you might have forgotten to add core-datasets folder (so just `git add core-datasets`, then `git commit -m 'added core-datasets content`, then git push)

**Review by scottyhq — APPROVED** [review]

**Comment by scottyhq** [comment]
thanks @meganmason ! you can see a preview of your changes with the link above. happy to merge this as is if you'd like, or we can keep it open to keep pushing changes to.

**Comment by meganmason** [comment]
Thanks for the preview! We can leave it open since there is a lot more to add. I likely won't be able to generate content for a week or so if that's an issue.

**Comment by jomey** [comment]
@scottyhq - This branch has still the old gitaction config, where builds fail with broken external links. Can you please merge this one? It takes admin privileges to do that

**Comment by scottyhq** [comment]
I just rebased this, @meganmason , I can either merge as is and you can open a new pull request to modify content, or you can run `gh pr checkout 22` and continue adding modifications. Preference?

**Pull request merged**